.github/workflows/package.yaml

```diff
@@ -3,107 +3,168 @@ name: Build binaries and create a draft release
 on:
   push:
     tags: ['*']
+  workflow_dispatch:
+    inputs:
+      TAG:
+        description: Docker CLI tag to build
+        required: true
+        default: v20.10.17
 
 defaults:
   run:
     shell: bash
 
-env:
-  DOCKER_CLI_REF: v20.10.17
-
 jobs:
   build:
     strategy:
       fail-fast: false
     runs-on: ubuntu-latest
 
+    defaults:
+      run:
+        working-directory: cli
+
     steps:
+    - name: Set Docker CLI tag from tag
+      if: startsWith(github.event.ref, 'refs/tags/')
+      run: |
+        DOCKER_CLI_REF="${{ github.event.ref }}"
+        echo "DOCKER_CLI_REF=${DOCKER_CLI_REF#refs/tags/}" >> "${GITHUB_ENV}"
+      working-directory: .
+
+    - name: Set Docker CLI tag from inputs
+      if: inputs.TAG
+      run: echo "DOCKER_CLI_REF=${{ inputs.TAG }}" >> "${GITHUB_ENV}"
+      working-directory: .
+
     - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@v1
+      uses: docker/setup-buildx-action@v2
 
-    - uses: actions/checkout@v2
+    - name: Check out build scripts
+      uses: actions/checkout@v3
+      with:
+        persist-credentials: false
+        path: build-scripts
```

Comment:
As discussed in slack, I'd prefer to call this `rancher-desktop-docker-cli` so it's that much more obvious what's going on.